README.md

```diff
@@ -78,7 +78,7 @@ Basically for most use cases, Entity is the entity/model/array of hashes you wis
 ```ruby
 class Post::Finder < Trailblazer::Finder
   # Without defining an ORM everything defaults to dealing with hash objects
-  adapters :ActiveRecord
+  adapter "ActiveRecord"
```

Comment:
only string or you accept symbols as well? maybe we should handle both if not handled yet